book/website/community-handbook/communication-channels/slack-start-guide.md

```diff
@@ -0,0 +1,110 @@
+
+**Table of content**
+
+1. [Configuring your account](#1-Configuring-your-account)
+2. [Notification settings](#2-Channel-and-Notification-settings)
+3. [Communicating with others](#3-Communicating-with-others)
+
+## 1. Configuring your account
+
+### PROFILE
+
+* **Your username** – please use a name by which others will be able to identify you. Your email address is visible to others.
+* **Your avatar** – please add a profile picture or other identifier that you’re comfortable with so that others can associate your posts with you more easily.
+* **Your time zone** – this determines when Slack will notify you and can also help others know whether it’s a reasonable time of day to get in touch.
+
+Here is a guide to help you [edit your profile]( https://slack.com/intl/en-ca/help/articles/204092246-Edit-your-profile).
+
+### STATUS UPDATES
+
+Status updates can be a useful way to let others know your availability as your status will be visible to everyone in the Slack group. 
+
+To update your status, click your name in the top left of the screen and select `update status`. You can select when you want your status update to be removed by stipulating a timeframe from the “clear after” dropdown.
+You can clear a status update at any time by clicking on your name and selecting `clear status` from the dropdown.
+
+## 2. Channel and Notification settings
+
+### How to find, join and create channels
+
+By default, you’re automatically added to the main channels that everyone belongs to (for example, #general, #welcome, #introductions & #random).
+
+You will be able to create additional channels to facilitate structured conversation. You are free to create as many channels as you would like. However, channels dedicated to projects should be made public (not private) so people can collaborate or help you.
+
+### Configure your notifications
+Slack notifications are great, but they may bother you when you try to focus on your project. There are a lot of options for you to determine how and when you are informed about content – and at what level of granularity. 
+
+#### For the overall group
+1. Click on your `name` at the top left of the page and select “`preferences`” from the dropdown
+1. In the Notifications section you have options which include:
+    * Set times when you do not want to be disturbed by notifications
+    * Enable desktop notifications
+    * Choose to be alerted when certain keywords are mentioned
+
+#### Channel by channel
+1. When you’re in a channel, click the “`i`” icon at the top right. Choose “`more`” and then click “`notifications`.”
+    * Here you have the option to ignore any @channel messages, or mute the channel entirely.
+
+#### Following a specific thread
+If another group member posts something of particular interest you can choose to follow that thread. Find how to reply in thread in [Section 3](#3-Communicating-with-others).
```

Comment:
If another group member posts something of particular interest you can choose to follow that thread. Learn how to reply in thread in [Section 3](https://github.com/the-turing-way/the-turing-way/pull/3380/files#3-Communicating-with-others).
```suggestion
If another group member posts something of particular interest you can choose to follow that thread. Find how to reply in thread in [Section 3](#3-Communicating-with-others).
```

Reply:
Hi @JFormoso 

This is the same as the above reply ;) 

The only two files that this PR changes and is able to discuss are:
- book/website/collaboration/inclusive-events/social-activities.md
- book/website/collaboration/inclusive-events.md